bin/ch/WScriptJsrt.cpp

```diff
@@ -188,10 +188,9 @@ JsValueRef WScriptJsrt::LoadScriptHelper(JsValueRef callee, bool isConstructCall
     else
     {
         const wchar_t *fileContent;
-        char *fileName;
+        char *fileName = (char*) "script.js";
```

Comment:
Yup, this was the existing behavior- it looks like if a path is not provided, the default is script.js